### DISCUSSION TITLE
Tooltips not working on Linux

### DISCUSSION THREAD

**Discussion by caseyjp11** [X2CommunityCore/xcom2-launcher#388] [Help]
**AML Version:** 
The latest production AND the same issue persist with both betas.
AML is installed under Arch Linux+Steam with no problems other than the application crasher that occurs for all platforms once the launch button is used.

If no tooltips are encountered, the application operates normally and can launch the game.

**Description**
The attached picture shows the issue. Initially, the tooltips work. After that, the picture attached shows what occurs.
The tooltip goes black and the app becomes unresponsive. Killing the session via Steam and restarting the game clears the problem.
This happens on all proton versions including experimental and ALL "GE" versions. (X11/Wayland = same problem.)
The 8-32 GE release was the last time this wasn't an issue.
Other proton games using tooltips + the same proton versions do NOT exhibit this problem.

**Attachments**
![AML text box crash](https://github.com/X2CommunityCore/xcom2-launcher/assets/11329574/55c00a25-a593-4228-876c-02d994ad81a4)

* *Consider attaching the `AML.log` file from the AML folder*
* *Consider attaching the `settings
* *If applicable, add screenshots to help explain your problem.*

**Comment by RevZero** [comment]
There is not much, we can do about that. AML uses the standard ToolTip control. I am not familiar with the internal workings of Proton, only that it is based on Wine. I haven't heard of any Wine users with similar problems, so maybe a Proton issue.

**Comment by caseyjp11** [comment]
The report is up more for documentation purposes than anything else.
I filed reports when the glitch happened against Steam/Proton/GE applications.

Murphy's Law says: The bug report was rejected as an APPLICATION SPECIFIC ISSUE. LOL.

I am not surprised that this is a 'rare' problem. The ONLY way to install the AML is to follow this procedure via a 3-year-old thread:
[(https://www.reddit.com/r/xcom2mods/comments/rhjeby/you_can_run_aml_on_linux_proton_howto_included/)](url)

The application still launches the game and mods successfully aside from the tooltip issue. I've been running WotC via AML on Arch Linux/Steam for over two years at this point and still do as I am a complete XCom junkie.

(Also, just FYI, but Proton == wine. The open-source project has become massive, but Proton is the wine variant that has been designed specifically to run with Steam. Valve, Codeweavers, and the general wine community are one big family and the patches applied are upstreamed to the wine project.)

You probably already know this, but as I said, the thread is now more of an information post, so there it is.
